tests/microbenchmarks.py

```diff
@@ -4,19 +4,24 @@
 
 from utils import *
 
+
 def test_small_integers(test_case):
     print("Running", "test_small_integers")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
         for i in range(100):
             for j in range(100):
                 x = -5
                 for k in range(-5, 257):
                     x = x + 1
-        """), timed=True)
+        """),
+        timed=True)
+
 
 def test_booleans(test_case):
     print("Running", "test_booleans")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils